main.py

```diff
@@ -1,9 +1,18 @@
+"""Flask app."""
 from flask import Flask
-app = Flask(__name__)
 
-@app.route("/")
+APP = Flask(__name__)
```

Comment:
this being capitals is a bit ugly... we could make an exception in the pylintrc

Reply:
Added to to do list

Reply:
> this being capitals is a bit ugly... we could make an exception in the pylintrc

pylintrc file generated, changed one bit of the code; from "UPPER_CASE" to "any":

"# Naming style matching correct constant names
const-naming-style=any"